Update custom cards, add custom badges and custom card features (#2258)
home-assistant/developers.home-assistant: docs/frontend/custom-ui/custom-card-feature.md

---
title: "Custom card feature"
---

> Some dashboard cards have support for [features](https://www.home-assistant.io/dashboards/features/). These widgets add quick controls to the card. We offer a lot of built-in features, but you're not just limited to the ones that we decided to include in Home Assistant. You can build and use your own using similar way than defining [custom cards](/docs/frontend/custom-ui/custom-card).

Author: ```suggestion
Some dashboard cards have support for [features](https://www.home-assistant.io/dashboards/features/). These widgets add quick controls to the card. We offer a lot of built-in features, but you're not just limited to the ones that we decided to include in Home Assistant. You can build and use your own in the same way as defining [custom cards](/docs/frontend/custom-ui/custom-card).
```
